src/Questionnaire/Questionnaire.test.js

```diff
@@ -0,0 +1,30 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+import Questionnaire, { countYes } from './Questionnaire'
+
+describe('The questionnaire component', () => {
+  it('renders correctly', () => {
+    const tree = render(<Questionnaire />)
```

Comment:
I think you only need the property `container` for the snapshot, not the whole object.
i.e.
```js
const { container } = render(...)
```